sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -70,6 +70,10 @@ object RapidsPluginUtils extends Logging {
   private val EXECUTOR_GPU_AMOUNT_KEY = "spark.executor.resource.gpu.amount"
   private val SPARK_MASTER = "spark.master"
 
+  private val pluginPropsNames = Seq("rapids-4-spark-sql_2.12", "rapids-4-spark-sql_2.13")
```

Comment:
My concern is still that the complexity is increasing but we are still not covering the jars from prior releases.

Let us look things that have always been there in 

```bash
TZ=UTC ~/dist/spark-3.3.0-bin-hadoop3/bin/spark-shell --driver-class-path=$PWD/dist/target/parallel-world  --jars dist/target/rapids-4-spark_2.12-23.12.0-SNAPSHOT-cuda11.jar --conf spark.plugins=com.nvidia.spark.SQLPlugin
``` 

```scala
scala> ShimLoader.getShimClassLoader().getResources("com/nvidia/spark/SQLPlugin.class").asScala.toSet
res1: scala.collection.immutable.Set[java.net.URL] = Set(file:/home/gshegalov/gits/NVIDIA/spark-rapids.worktrees/speedupBuildInfo/dist/target/parallel-world/com/nvidia/spark/SQLPlugin.class, jar:file:/home/gshegalov/gits/NVIDIA/spark-rapids.worktrees/speedupBuildInfo/dist/target/rapids-4-spark_2.12-23.12.0-SNAPSHOT-cuda11.jar!/com/nvidia/spark/SQLPlugin.class)

scala> ShimLoader.getShimClassLoader().getResources("ai/rapids/cudf/Table.class").asScala.toSet
res2: scala.collection.immutable.Set[java.net.URL] = Set(file:/home/gshegalov/gits/NVIDIA/spark-rapids.worktrees/speedupBuildInfo/dist/target/parallel-world/ai/rapids/cudf/Table.class, jar:file:/home/gshegalov/gits/NVIDIA/spark-rapids.worktrees/speedupBuildInfo/dist/target/rapids-4-spark_2.12-23.12.0-SNAPSHOT-cuda11.jar!/ai/rapids/cudf/Table.class)
```

Once we got duplicate URLs we can probe for varying property files to dump additional diagnostics.

Reply:
Nice catch, now I filter out 'test' from the properties URL to make it cover prior releases, since we are not going to change properties names in this PR.